diffblue-base.json

```diff
@@ -0,0 +1,16 @@
+{
+  "phaseBase": {
+    "preferDepsJar": true,
+    "context-include": [
+      "org.springframework.mvc.extensions.ajax.AjaxUtils",
```

Comment:
This prefix is missing the `.` at the end. Not that it really matters.

Reply:
I wanted to be careful with `.` at the end of class names, as they might be followed by a `$` instead. It doesn't look like it would be the case here though.

Reply:
Oh, oops, if that's a class name then I agree it shouldn't end with a `.`.